src/components/PoiItem.jsx

```diff
@@ -28,8 +32,16 @@ const PoiItem = React.memo(
               <ReviewScore reviews={reviews} poi={poi} inList={inList} />
             </div>
           )}
-          <div className="poiItem-subclassAndHours">
-            <div className="poiItem-subclass">{subclass}</div>
+          <div className="poiItem-subclassStarsAndHours">
+            <span className="poiItem-subclass">{subclass}</span>
+            {!inList && subclass && hasStars && '\u00A0⋅\u00A0'}
```

Comment:
Not your fault, but all these rules about list or not are too complicated…
Maybe it'll be worth splitting the list/popup card again, or we can try to convince the designers to always display everything the same way.